### ISSUE TITLE
Event handler ignore cancel doesn't work

### ISSUE THREAD

**Issue by AzonFR** [btk5h/skript-mirror#154]
On my server, i have Factions and WorldGuard and when i use this script:
https://pastebin.com/D3hMASxX
I can listen to the event but when i'm in warzone (build is disabled) or in a region where build is disabled, the script don't work and i don't know why ?

**Server information**
- skript-mirror: 0.19.1
- Skript: 2.2-dev36
- Bukkit: PaperSpigot 1.8.8
- Java: 8
- OS: Ubuntu 18.04

**Comment by AzonFR** [comment]
so @btk5h , do you have an idea ? ...

**Comment by btk5h** [comment]
Not enough information to reproduce, can you post more information (plugin versions, setting up regions, etc.) on how to replicate this on a fresh server?

**Comment by AzonFR** [comment]
I'll try to replicate this on a fresh server, i tell you

**Comment by AzonFR** [comment]
Hey, i just try on a 1.12.2 PaperSpigot version, clean install, skript version 2.2-dev36
I didn't work, when the event isn't cancelled, it broadcast "Azon_FR" but when the event is cancelled (by WorldGuard here) there is no broadcast
Code: https://pastebin.com/NkG4piZH
So i try with the latest version of Skript (https://github.com/SkriptLang/Skript/releases) (2.5) and still the same problem
Versions: https://gyazo.com/3e43f4d26785bc261789a99c9b1968c9
Issue: https://gyazo.com/d3552506bb68babb12b04a5759e92d6c
There is no error in the console

**Comment by AzonFR** [comment]
bump

**Comment by AzonFR** [comment]
bump :/

**Comment by btk5h** [comment]
I have my suspicions on what could be causing this, does your script work if you remove the "all" keyword?

**Comment by AzonFR** [comment]
Yes it work but when the event is cancelled, it don’t work. So it’s the
same as if i add "all"

Le jeu. 18 juin 2020 à 20:12, Bryan Terce <notifications@github.com> a
écrit :

**Comment by AzonFR** [comment]
Yes it work but when the event is cancelled, it don’t work. So it’s the
same as if i add "all"

**Comment by btk5h** [comment]
Anyone looking into this issue might want to look into making sure the ignoreCancelled flag is being set properly
https://github.com/btk5h/skript-mirror/blob/6e4e8bf7a640256baca2bfbe877445a378fe8319/src/main/java/com/btk5h/skriptmirror/skript/reflect/EvtByReflection.java#L126
and make sure the ignoreCancelled flag actually works when registering an event.
https://github.com/btk5h/skript-mirror/blob/6e4e8bf7a640256baca2bfbe877445a378fe8319/src/main/java/com/btk5h/skriptmirror/skript/reflect/EvtByReflection.java#L100

**Comment by AzonFR** [comment]
Thanks for help, so i have to wait for an update?

**Comment by AzonFR** [comment]
up ^^
